x/ecocredit/server/marketplace/update_sell_orders.go

```diff
@@ -119,8 +121,8 @@ func (k Keeper) applySellOrderUpdates(ctx context.Context, order *api.SellOrder,
 				return err
 			}
 			order.Quantity = update.NewQuantity
-			event.NewQuantity = update.NewQuantity
 		}
+		event.NewQuantity = update.NewQuantity
```

Comment:
As mentioned above, I think we should remove this from the event and only include the seller address and the sell order id when emitting an event for updating the sell order.